tests/Integration/Behaviour/Features/Scenario/Title/title_management.feature

```diff
@@ -11,21 +11,15 @@ Feature: Title management
     When I add a new title "3rdTitle" with the following properties:
       | name    | Third Title    |
       | type    | Other          |
-#    Then title "thirdTitle" name should be "Normandy"
-#    Then title "thirdTitle" country should be "United titles"
-#    Then title "thirdTitle" zone should be "Europe"
-#    And title "thirdTitle" should be enabled
-#
-#  Scenario: Editing title
-#    When I edit title "thirdTitle" with following properties:
-#      | name    | Britain       |
-#      | enabled | false         |
-#      | country | Italy         |
-#      | zone    | South America |
-#    Then title "thirdTitle" name should be "Britain"
-#    Then title "thirdTitle" country should be "Italy"
-#    Then title "thirdTitle" zone should be "South America"
-#    And title "thirdTitle" should be disabled
+    Then the title "3rdTitle" name should be "Third Title"
+    And the title "3rdTitle" gender type should be "Other"
+
+  Scenario: Editing title
+    When I edit title "3rdTitle" with following properties:
+      | name    | Fourth Title   |
```

Comment:
could I ask to test multilang?
Plenty of examples under product domain, but you would basically need to add couple language assertions steps in background:
```
    And language "en" with locale "en-US" exists
    And language "fr" with locale "fr-FR" exists
    And language with iso code "en" is the default one
```

and then when editing/creating use it like this to provide localized langs
```suggestion
      | name[en-US]  | Fourth Title             |
      | name[fr-FR]    | Fourth Title  french  |
```

and in the context file you can use $this->localizeByRows($tableNode) and you will get prepared names indexed by id

Reply:
Fixed